src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -241,7 +252,7 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
                         openWeb.setTooltip(new Tooltip(Localization.lang("Open URL or DOI")));
                         openWeb.setOnMouseClicked(event -> {
                             String url = entry.entry().getDOI().flatMap(DOI::getExternalURI).map(URI::toString)
-                                              .or(() -> entry.entry().getField(StandardField.URL)).orElse("");
+                                                .or(() -> entry.entry().getField(StandardField.URL)).orElse("");
```

Comment:
Both the old and the new ident look strange. Was this the result of auto format?